.ci/ldif/user1.example.com.ldif

```diff
@@ -0,0 +1,5 @@
+dn: uid=user1,dc=example,dc=com
```

Comment:
This user is in some of the tests expected to be named ```anotherUser``` and causes these tests to fail